components/Tabs/examples/supplemental/TabBarViewControllerExampleSupplemental.m

```diff
@@ -152,8 +152,8 @@ - (void)setupTabBarColors {
 - (nonnull NSArray *)constructExampleViewControllers {
   NSBundle *bundle = [NSBundle bundleForClass:[TabBarViewControllerExample class]];
   MDCButtonScheme *buttonScheme = [[MDCButtonScheme alloc] init];
-  buttonScheme.colorScheme = self.colorScheme;
-  buttonScheme.typographyScheme = self.typographyScheme;
+  buttonScheme.colorScheme = self.containerScheme.colorScheme;
```

Comment:
Should the container scheme be initialized in this implementation at some point?  Maybe in `init` or `viewDidLoad` or a similar method?

Reply:
It happens in init but it's in a different file, `TabBarViewControllerExample.m`. These examples are set up kind of weird it seems.